mino/minogrpc/server_test.go

```diff
@@ -525,9 +558,19 @@ func TestConnManager_Acquire(t *testing.T) {
 	require.Len(t, mgr.conns, 0)
 	require.Equal(t, 0, mgr.counters[dst.GetAddress()])
 
+	mgr.certs = fakeCerts{errLoad: fake.GetError()}
+	_, err = mgr.Acquire(fake.NewAddress(0))
+	require.EqualError(t, err, fake.Err("while loading distant cert"))
+
+	mgr.certs = certs
 	_, err = mgr.Acquire(fake.NewAddress(1))
 	require.EqualError(t, err, "certificate for 'fake.Address[1]' not found")
 
+	mgr.certs = fakeCerts{errLoad: fake.GetError(), counter: fake.NewCounter(1)}
+	_, err = mgr.Acquire(fake.NewAddress(0))
+	require.EqualError(t, err, fake.Err("while loading own cert"))
+
+	mgr.certs = certs
 	mgr.conns = make(map[mino.Address]*grpc.ClientConn)
 	mgr.certs.Delete(mgr.me)
 	_, err = mgr.Acquire(dst.GetAddress())
```

Comment:
Sorry for repeating myself, but I would split the failing cases in separate tests